editor/register_editor_types.cpp

```diff
@@ -129,6 +129,7 @@
 #include "editor/plugins/visual_shader_editor_plugin.h"
 #include "editor/plugins/voxel_gi_editor_plugin.h"
 #include "editor/register_exporters.h"
+#include "import/3d/scene_import_settings.h"
```

Comment:
```suggestion
#include "editor/import/3d/scene_import_settings.h"
```
(and moved up to be alphabetically ordered)